mmv1/third_party/terraform/services/container/resource_container_cluster.go.tmpl

```diff
@@ -5731,8 +5674,9 @@ func expandMaintenancePolicy(d *schema.ResourceData, meta interface{}) *containe
 				meo := exclusionOptions.([]interface{})[0].(map[string]interface{})
 				mex := exclusions[exclusion["exclusion_name"].(string)]
 				mex.MaintenanceExclusionOptions = &container.MaintenanceExclusionOptions{
-					Scope:           meo["scope"].(string),
-					ForceSendFields: []string{"Scope"},
+					Scope:             meo["scope"].(string),
+					EndTimeBehavior:   meo["end_time_behavior"].(string),
+					ForceSendFields:   []string{"Scope", "EndTimeBehavior"},
```

Comment:
EndTimeBehavior is Optional, does it need to be in ForceSendFields: ?

Reply:
It depends on the API's behavior for this field.

There seems to only be one valid value for the field, is it possible to unset that? Would that make sense?

Reply:
yes, the field has only one valid value, and it is possible to unset it. the behavior is: it is either unset, or set with that one single valid value.

for this case, does it need to be in ForceSendFields: ? i don't really know what does ForceSendFields: do, i guess that's also my question.